content/4_Gold/SP.mdx

```diff
@@ -358,7 +358,11 @@ public class test {
 
 <Warning>
 
-It's important to include `continue`. This ensures that when all edge weights are non-negative, we will never go through the adjacency list of any vertex more than once. Removing it will cause TLE on the last test of the above problem!
+It's important to include `continue`. This ensures that when all edge weights are non-negative, we will never go through the adjacency list of any vertex more than once. Removing it will cause TLE on the last test of the above problem! 
+
+The last testcase contains 100000 destinations and 149997 routes. City 1 has around 50000 routes to cities 2-50000. Cities 2-50000 have routes that go to City 50001. City 50001 has routes that go to Cities 50002-100000. Without the `continue`, after the program processes the routes from City 1, the priority queue will contain 49998 routes that point to City 50001. After getting the shortest route to City 50001 from these routes, it will move on to the cities 50002-100000 that can be reached from City 50001. However, the priority queue now has 49998 instances of these routes. As you can see, this runtime is worse than $\mathcal{O}(N^2)$, which results in a TLE. 
```

Comment:
> As you can see

It's not immediately obvious why, so many people might not see. I think you should include something like "this means that city 50001 will be processed 49998 more times"